generators/client/templates/angular/package.json.ejs

```diff
@@ -39,6 +39,7 @@
     "@fortawesome/free-solid-svg-icons": "5.1.0-8",
     "@ng-bootstrap/ng-bootstrap": "2.0.0",
     "bootstrap": "4.0.0",
+    "friendly-errors-webpack-plugin": "1.7.0",
```

Comment:
This should't be here. You already have it in dev deps

Reply:
Fixed